### ISSUE TITLE
can not use Tailwind CSS IntelliSense v0.10.1 in vscode

### ISSUE THREAD

**Issue by weykon** [bocarw121/tailwind-yew-cli#2]
there always display `[tailwind.config.js] Building...` in Tailwind CSS IntelliSense output

I already set in default 
```
/** @type {import('tailwindcss').Config} */
module.exports = {
	content: ['./src/**/*.{html,rs}', 'index.html'],
	theme: {
		extend: {},
	},
	plugins: [
		require('@tailwindcss/typography'),
		require('@tailwindcss/forms'),
	],
};
```

how should I refer to it ?

**Comment by bocarw121** [comment]
Hey, I'm just curious from the title it says you can't use intelliSense, are you having issues with the  auto complete for your html class attribute? 

As far as the Tailwind CSS IntelliSense output it should show `[tailwind.config.js] Building...` and once a rust or html file is changed it should show `[tailwind.config.js] File changed:` I tested it with your `tailwind.config.js` and didn't find any issues. Does you styles folder have an input.css and output.css and did you run the watcher command from the `tailwind_info.md`? Can you give me more details if possible and I will do my best to help you out.

Also, in the setup step did you choose the correct binary for your system?

**Comment by weykon** [comment]
```
Unhandled exception: ENOENT: no such file or directory, open './dist/.stage/output-304f6e8c2f88e387.css'
```
After I tried again, it was the log after `building` in extension output. I found `output-304f6e8c2f88e387.css` in my file directory, but its path was `./dist/output-304f6e8c2f88e387.css`, without `.stage` folder.

My mac is intel chip and I selected the mac x64. 

It is available in `html` file, but ./src/main.rs was not working. 

It is running two commands, `./tailwindcss -i styles/input.css -o styles/output.css --watch` and `trunk serve`

Could the lint dialog display while pressing SPACE in rust file ?

This is probably more related to this extension thing.

**Comment by bocarw121** [comment]
I am running with the same system as well and i did notice this `Unhandled exception: ENOENT: no such file or directory, open './dist/.stage/output-304f6e8c2f88e387.css'`. I will look into this issue

Yes it could display when I pressed space and I navigated to the Tailwind CSS IntelliSense to get the output. 

True, it could be an extension thing. 

Overall, does your tailwind compile and do the changes reflect in the UI? 

Thanks
